code/game/objects/items/storage/backpack.dm

```diff
@@ -471,7 +471,8 @@
 		slowdown = initial(slowdown)
 		atom_storage.locked = STORAGE_SOFT_LOCKED
 		atom_storage.display_contents = FALSE
-		atom_storage.close_all()
+		for(var/obj/item/storage/storage_item in get_all_contents_type(/obj/item/storage)) //close ui of this and all storage items(e.g. boxes) inside dufflebag
```

Comment:
why are you filtering for storage subtype?

Reply:
Because no other types can hold items, and we perform the close operation only on items like boxes

Reply:
that's completely incorrect, the only thing /datum/storage does is be a shortcut to applying the atom_storage datum. it can be used by any atom and is used by stuff like fedoras, shoes, storage modules etc

Reply:
`as anything` doesn't work. During map load apparently there are null items inside the duffle bag and it runtimes. 

`obj/item` should be a better type assumption to filter by. `get_all_contents_type()` can return a large number of items so filtering by this type is the best bet. I'll do that then

Reply:
![Screenshot (294)](https://github.com/tgstation/tgstation/assets/110812394/5e38ed31-c55e-48e8-8d78-25efd7e697e9)

I don't know what to do. i guess checking for null items is my only option

Reply:
done